tests/devapps/MauiMacAppWithBroker/MainPage.xaml.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace MauiMacAppWithBroker;
+
+using Microsoft.Maui.Controls;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Broker;
+using System;
+using System.Runtime.InteropServices;
+
+public partial class MainPage : ContentPage
+{
+	private bool inUse = false;
+
+	public MainPage()
+	{
+		InitializeComponent();
+	}
+
+	private async void OnACIACSClicked(object sender, EventArgs e)
+	{
+		
+		if (inUse)
+		{
+			await Application.Current.MainPage.DisplayAlert("Warning", "Please wait until previous operation finished", "OK").ConfigureAwait(false);
+			return;
+		}
+		inUse = true;
+
+		SemanticScreenReader.Announce(CACIACSBtn.Text);
+
+        PublicClientApplicationBuilder builder = PublicClientApplicationBuilder
+			.Create("04b07795-8ddb-461a-bbee-02f9e1bf7b46")
+			.WithRedirectUri("msauth.com.msauth.unsignedapp://auth")
+			.WithClientCapabilities(new[] { "cp1" })
```

Comment:
do you need `WithClientCapabilities` the request is not even passing any claims or testing the CAE flow

Reply:
Does it matter? It's just a test app. It's not enough to test teh full CAE flow, but it is enough to test that CP1 is going through.